README.md

```diff
@@ -27,6 +27,14 @@ cd caffe-android-lib
 ../builder.sh
 ```
 
+## Building with docker
```

Comment:
maybe we should move it before normal `building` section because with docker we don't need local clone recursive.

Reply:
Hmm got it, i will change here!